.changeset/purple-suns-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+'@neo4j-cypher/language-support': patch
+---
+
+Fix issue where syntax highlighting crashes on create constraint query
```

Comment:
I think we should leave this file out of the PR and same for #138. Because we have a patch with a different file name in both so I don't know what's the expected behaviour in that case